bendigobank_spec.rb

```diff
@@ -0,0 +1,54 @@
+require 'rspec'
+require 'nokogiri'
+require_relative 'account.rb'
+require_relative 'transaction.rb'
+require_relative 'bendigobank.rb'
+
+describe Bendigobank do
+
+  html_account = Nokogiri::HTML(File.read('/home/vlad/Documents/test_task/accounts.html'))
```

Comment:
Use a relative path. No need to hardcode your name in the project.
`File.read('accounts.html')`